deploy/prometheus/prometheus-alert-rules.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   groups:
     - name: ./clickhouse-operator.rules
       rules:
-        - alert: MetricsExporterDown
+        - alert: ClickHouseMetricsExporterDown
```

Comment:
please also do this replace in `./tests/test_mertics_alerts.py`
for consistent rename alerts